backend/src/api/overpass.py

```diff
@@ -71,13 +72,11 @@ def nearby_point(lat_long: Tuple[float, float], radius: int) -> List[Node]:
     })
 
     if not response.ok:
-        # TODO: What's a useful response here
-        return None
+        raise ExternalAPIError("Issue with external Overpass API")
 
     response_json = response.json()
     if "elements" not in response_json:
```

Comment:
When would this happen?

Reply:
If the api was down for some reason I think. I don't see it ever happening really, but I think it's a good thing to include just in case